en/releases/main.md

```diff
@@ -39,7 +39,8 @@ TBD ...
 
 ### Simulation
 
-- TBD
+- [Gazebo] Support for [Advanced Plane](../gazebo_vehicles.md#advanced-plane), a simulated fixed-wing vehicle that provides better aerodynamic simulation than the regular plane.
+  Added to PX4 in [PX4-Autopilot#22167](https://github.com/PX4/PX4-Autopilot/pull/22167) and [gz-sim#2185](https://github.com/gazebosim/gz-sim/pull/2185) (advanced lift drag plugin).
 
```

Comment:
@frede791 I'm trying to be a bit more organized about release notes for the next release, so adding some updates as we go. Can you edit this to capture what you think is important? Note that you can see https://docs.px4.io/main/en/releases/1.14.html for a model of what is happening here.

Reply:
fixed the path but the rest is good I think